src/docs/01-Components/06-Feedback/04-spinner.md

```diff
@@ -28,4 +28,12 @@ secondaryKeywords: indicator spin interaction rotate
 # HTML Guidelines
 
 # UX and Design Guidelines
+Try to always keep spinners in page and not in overlays.
+If a spinner is triggered by a button, place the spinner in the button, and disable the button while the spinner is visible.
+
+If only a portion of a page is displaying new content or being updated, place the spinner in that part of the page.
+
+If you are unsure where to place the spinner, place it where you want the user's attention to be when loading is finished.
+Only show a spinner if the expected wait time is more than a second.
+Add a minimum of 100s of delay to mitigate lots of unnecessary spinners showing up at the same time.

```

Comment:
100 seconds might be overdoing it ;) On a more serious note the last line is a bit difficult to understand. Why would adding a delay prevent multiple spinners at the same time?

Reply:
Good catch! I have removed the last confusing part and updated PR